recipes/laszip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "laszip" %}
+{% set version = "3.2.2" %}
+{% set sha256 = "0b2e5df3b4f464da9331dfc7bd455044a656be0f2251e50be5f087417ad41876" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc14  # [win]
+  skip: True  # [py!=36]
+
+requirements:
+  build:
+    - cmake
+  run:
+    - vc 14  # [win]
+
+test:
+  commands:
+    - test -d $PREFIX/include/laszip  # [not win]
+    - if not exist %LIBRARY_INC%\\laszip exit 1  # [win]
+
+about:
+  home: https://www.laszip.org
+  license: LGPL-2.0
+  license_family: LGPL
+  license_file: COPYING
```

Comment:
@chambbj, this indicates that the license file is copied from SRC_DIR/COPYING which is from the tarball. You don't need the file in the recipe folder.

Reply:
Oh, of course. Thanks for clarifying.